demos/default/next.config.js

```diff
@@ -71,7 +71,7 @@ module.exports = {
   },
   // https://nextjs.org/docs/basic-features/image-optimization#domains
   images: {
-    domains: ['raw.githubusercontent.com', 'upload.wikimedia.org'],
```

Comment:
Removed `'upload.wikimedia.org'`, as it was added a few weeks ago to prevent the `/images` page in the default demo from breaking due to Next.js throwing when the host is not supported for an image.